### PR TITLE
Portfolio setup(Desktop version)

### PR DESCRIPTION
**Pull request by collinsmezie**
Desktop version of project portfolio, with completed projects , project description, developer contact and form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -225,7 +253,11 @@ <h2>Frameworks</h2>
         <h2>Skills</h2>
         <span class="fas fa-angle-right"></span>
       </div>
+      </div>
     </div>
+
+
+
     <div class="form-section">
```
Comment (Deepakdanger) [inline index.html:262]: - [ ] In **Desktop version** ,The `Contact me` section should have same design as original website. Kindly fix the `top-left design` and background image of the `contact me` section. **Hint** You can do that by using `border radius` for design.
- Your website 
gerge

- Original website
gwg

File: index.html [hunk 2]
```diff
@@ -163,11 +171,13 @@ <h2>Topic</h2>
...
             </li>
           </ul>
+          <button type="button" class="big-btn">See Project</button>
         </div>
-        <button type="button" class="big-btn">See Project</button>
+      
       </div>
     </div>
-    <div class="intro">
+    <div class="About">
```
Comment (Deepakdanger) [inline index.html:180]: - [ ]  In **Desktop version**, The `About Myself` section should have the same design as the original website. Kindly fix the top-right design. Hint You can do that by using border-radius for design.
- Your website  
gergsas

- Original website
ngfbdf

File: index.html [hunk 3]
```diff
@@ -27,11 +27,19 @@
...
             ><img src="media/Union.svg" alt="menu icon" class="hamburger"
           /></a>
+          <ul id="nav-links">
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+          </ul>
         </nav>
       </header>
       <div class="intro">
```
Comment (Deepakdanger) [inline index.html:37]: - [ ] In **Desktop version**, The `top section` should have the same design as the original website. Kindly fix the bottom-left design. Hint You can do that by using `border-radius` for design.
- Your website 
ssdg
- Original website 
ffdvfv

File: index.html [hunk 4]
```diff
@@ -62,42 +70,41 @@ <h1>I'm Chime Glad to see you!</h1>
 
     <div id="projects-section">
```
Comment (Deepakdanger) [inline index.html:72]: - [ ] Change the `background color` of this section as in the original website, so the above-implemented design will be visible.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @collinsmezie, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 5]
```diff
@@ -251,3 +280,193 @@ textarea {
   background-color: #2230d2;
   color: #fff;
 }
+
+@media (min-width: 600px) {
```
Comment (ReshyResh) [inline styles.css]: - [ ] Please make sure the range of the desktop view is between `992px` and `1100px`, as per requirement.

File: styles.css [hunk 6]
```diff
@@ -251,3 +280,193 @@ textarea {
...
+    display: inline;
+  }
+
+  .paragraph {
+    width: 920px;
+    height: 98px;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: left;
+  }
```
Comment (ReshyResh) [inline styles.css:347]: - [ ] Giving the content fixed sizes makes it non responsive, and the content will go out of it's boundaries when resized, as in the picture below. Please solve this issue by making the content width dynamic and not fixed.

File: styles.css [hunk 7]
```diff
@@ -251,3 +280,193 @@ textarea {
...
+  .hamburger {
+    display: none;
+  }
+
+  .intro {
+    display: flex;
+    flex-direction: column;
+    width: min-content;
+    margin: 10em 0 0 10em;
+  }
```
Comment (ReshyResh) [inline styles.css:329]: - [ ] Please add some padding to these sections to better match the template.

File: styles.css [hunk 8]
```diff
@@ -251,3 +280,193 @@ textarea {
...
+  .About {
+    display: flex;
+    justify-content: space-between;
+    background: white;
+    width: 100vw;
+    padding: 100px 60px;
+    border-top-right-radius: 100px;
+    position: relative;
+    top: -120px;
+  }
```
Comment (ReshyResh) [inline styles.css:432]: - [ ] Please make sure the mobile version doesn't change when coding the desktop one. Currently, the mobile version is stuck to the left side of the screen, kindly fix that.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @collinsmezie , 

### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @collinsmezie , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations on the hard work, now you can take a rest for the weekend! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**